mem-store.js

```diff
@@ -1,120 +1,162 @@
 /* Copyright (c) 2010-2015 Richard Rodger, MIT License */
```

Comment:
@rjrodger @geek Since we now can have multiple lines do we need a multiline comment to cover this here? Can it be taken out in favour of a link the the license file?